gaia-header.js

```diff
@@ -761,9 +755,11 @@ module.exports = component.register('gaia-header', {
   }
 
   [action=close] .action-button:before { content: 'close' }
-  [action=back] .action-button:before { content: 'back' }
   [action=menu] .action-button:before { content: 'menu' }
 
+  [action=back]:-moz-dir(ltr) .action-button:before { content: 'back' }
+  [action=back]:-moz-dir(rtl) .action-button:before { content: 'forward' }
```

Comment:
When inspecting the DOM with Firefox Nightly, it seems that `-moz-dir(ltr)` is always applied, even when the whole document has an RTL direction. Do you have any clue or workaround to suggest?